ebean-core/src/main/java/io/ebeaninternal/server/deploy/BeanProperty.java

```diff
@@ -652,6 +652,9 @@ public void setTenantValue(EntityBean entityBean, Object tenantId) {
    */
   public void setValue(EntityBean bean, Object value) {
     try {
+      if (!beanHasProperty(bean)) {
```

Comment:
Technically I don't think this is needed on `setValue()` but instead on `setValueIntercept()` only

Reply:
Yes, setValueIntercept should be enough in both cases